packages/plugins/RSS3/src/SNSAdaptor/hooks/useDonations.ts

```diff
@@ -1,10 +1,8 @@
+import { RSS3 } from '@masknet/web3-providers'
 import { useAsync } from 'react-use'
-import type { AsyncState } from 'react-use/lib/useAsync'
-import { RSS3, RSS3BaseAPI } from '@masknet/web3-providers'
 
-export function useDonations(address: string): AsyncState<RSS3BaseAPI.Collection[] | undefined> {
+export function useDonations(address: string) {
     return useAsync(async () => {
-        const response = await RSS3.getDonations(address)
-        return response
+        return RSS3.getDonations(address)
```

Comment:
Does it necessary to query with a `chainId`?

Reply:
It's not according to the original code.